test/framework/namespace.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
IMHO the file name should be adjusted so it won't be confused with general namespace operations. What about for example `namespace_history.go`?

Reply:
Or as a second option to move namespace functionality here as mentioned by @radtriste

Reply:
Actually, in my first implementation, I named this script as "namespace_history.go", but then I changed it to just "namespace.go" because I wrote the methods OnNamespacePostCreated and OnNamespacePostDeleted in a way that can be refactored to do more things if needed in the future apart from just keeping this history log file. 

I would prefer to move the CreateNamespace and DeleteNamespace from kubernetes to here instead as @radtriste suggested. Are you ok with?

Reply:
sure, fine for me
Please move also `IsNamespace` and `OperateOnNamespaceIfExists` to have it all in one place.

Reply:
Done